docs/nodes/full-consensus-node.md

```diff
@@ -22,6 +22,8 @@ Full Consensus Nodes:
 * Disk: **250 GB SSD Storage**
 * Bandwidth: **1 Gbps for Download/1 Gbps for Upload**
 
+**Note**: Running a full consensus node requires significant storage capacity to store the entire blockchain history. As of the latest recommendation, it is advisable to have at least 250 GB of SSD storage for a Celestia full consensus node. Please ensure that your storage meets this requirement to ensure smooth syncing and operation of the node.
```

Comment:
[not blocking] IMO this seems redundant with the line above.

> * Disk: **250 GB SSD Storage**

so I don't think the addition in this PR is necessary but defer to docs owners

Reply:
good call. apologies for the confusion here. i believe that the original discussion resulted in:
- recommended storage: 250gb
- recommended storage without pruning (which would live in the "note": 500gb, context [here](https://github.com/celestiaorg/docs/pull/641#discussion_r1164568713)

Reply:
so the note should include the recommendation to use more storage if not using pruning, thank you for the PR @AnmolSirola 🙌